parser/ParseBlockProperties.hpp

```diff
@@ -0,0 +1,218 @@
+/**
+ *   Copyright 2011-2015 Quickstep Technologies LLC.
```

Comment:
Since this is a new file, it shouldn't have the Quickstep Technologies copyright, only the new copyright for Pivotal (also, the year should by 2016, or 2015-2016 if you also worked on it last year).